README.md

```diff
@@ -138,12 +138,18 @@ The integration program offers a bunch of commands, consult the `Use Cases` sect
 3. run the script
 
     ```sh
-    sbt capitaliq/run
+    sbt integrations/run
     ```
 
 ## Packaging
 
 creating a fat jar
 ```
-sbt capitaliq/assembly
+sbt integrations/assembly
+```
+
+## Testing
+
+```
+sbt integrations/test
```

Comment:
CI config uses just `sbt test` for testing. Please check if it runs tests in all subprojects and if so change readme.

Reply:
Well, there is only one subproject, so this runs all of the tests 😄